package/MDAnalysis/analysis/contacts.py

```diff
@@ -492,7 +514,7 @@ def timeseries(self):
 
 def _new_selections(u_orig, selections, frame):
     """create stand alone AGs from selections at frame"""
-    u = MDAnalysis.Universe(u_orig.filename, u_orig.trajectory.filename)
+    u = u_orig.copy()
```

Comment:
This technically changes the behaviour of q1q2, you could consider a case where `u` a trajectory which has been transferred in-memory and offset by a certain value (let's say you've shifted all the atomic positions by 10 A). By doing a copy on that object you now have a new trajectory that has the same shift in the atomic positions, whilst the old code would have led to a brand new universe that doesn't have this shift in atomic positions.

Whilst the copy behaviour is the one we are likely to want, introducing new changes that break behaviour without announcing it to users is a bad idea, because _someone_ might actually be relying on this behaviour (even though it's broken).

I realise @lilyminium pointed to fixing this in the original issue, but I don't agree that we should make this without warning users beforehand. I suspect further discussion will be needed here, so in the interest of saving time I would ask that we just not address this at this time and revisit it in the original issue before / fix it in a future PR.

Reply:
I see your point. From what you say, it does sound like it would be best to make a separate issue/PR out of it, and revert it back to original here. So should I go ahead and create a new issue for it and reference it here (as well as in #2666), or leave it to core developers to raise it?